src/types/Answers.ts

```diff
@@ -1,9 +1,11 @@
 import { type UUID } from 'crypto'
+import { type User } from './Users'
 
 export interface Answer {
   answer_id: UUID
   asker_id?: UUID
```

Comment:
This is a completely optional change, but it is relatively common to put **optional** fields last in an interface (to help with readability). Feel free to change if you want, no pressure though since it's really just a preference thing